### ISSUE TITLE
Custom error message for dependentValidate error

### ISSUE THREAD

**Issue by Sheikh566** [shystruk/csv-file-validator#114]
When `dependentValidate()` fails, it sets the error message from `validateError()` function given. But instead it should have its own error handler `dependentValidateError()` to show more precisely what is wrong.

**Comment by Sheikh566** [comment]
If this is a valid feature request. Please assign it to me

**Comment by shystruk** [comment]
go ahead and add updates @Sheikh566 
thanks for contributing

**Comment by Sheikh566** [comment]
@shystruk I am unable to run test. Getting this error:
```
node:internal/modules/cjs/loader:988
  throw err;
  ^

Error: Cannot find module 'D:\Study\Self-Learn\OpenSource\csv-file-validator\node'
    at Module._resolveFilename (node:internal/modules/cjs/loader:985:15)
    at Module._load (node:internal/modules/cjs/loader:833:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v18.10.0
----------|----------|----------|----------|----------|-------------------|
File      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
----------|----------|----------|----------|----------|-------------------|
All files |        0 |        0 |        0 |        0 |                   |
----------|----------|----------|----------|----------|-------------------|
```
Do you know how I can fix it?

**Comment by shystruk** [comment]
@Sheikh566 can you please push your branch and I will check what the issue is with tests

**Comment by Sheikh566** [comment]
@shystruk Can you add this repo to hacktoberfest 2023?

**Comment by shystruk** [comment]
@Sheikh566 share the link please

**Comment by Sheikh566** [comment]
@shystruk just add hacktoberfest tag to the repo

**Comment by shystruk** [comment]
closed in #129
